### PR TITLE
This should be revealing...

### PR DESCRIPTION
**Pull request by nirvn**
@m-kuhn, :santa:

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b0b9a1f8431d8b171165c48de252f431405dd86d:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-1ca3b402bfd8f4df0c53239f525af6a3469de610-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-1ca3b402bfd8f4df0c53239f525af6a3469de610-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-1ca3b402bfd8f4df0c53239f525af6a3469de610-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-1ca3b402bfd8f4df0c53239f525af6a3469de610-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b0b9a1f8431d8b171165c48de252f431405dd86d:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-de2ab853d87f838e96e5563602e27cd056c9a864-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-de2ab853d87f838e96e5563602e27cd056c9a864-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-de2ab853d87f838e96e5563602e27cd056c9a864-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-de2ab853d87f838e96e5563602e27cd056c9a864-x86.apk)

**Comment by m-kuhn** [comment]
🎄 🎁

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4268a5f7a2b696b5eeb106580e906bf0dd6e2625:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-7f1ac025df3ebdb9012b9afcd30c7b215d61ab70-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-7f1ac025df3ebdb9012b9afcd30c7b215d61ab70-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-7f1ac025df3ebdb9012b9afcd30c7b215d61ab70-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-7f1ac025df3ebdb9012b9afcd30c7b215d61ab70-x86.apk)

**Comment by nirvn** [comment]
Installed and briefly tested, working well.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0605dbb7f91897699afd71714e75c6c79bdf9d8a:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-2347306bdf11e71d3b103785c1f8574bc23f977a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-2347306bdf11e71d3b103785c1f8574bc23f977a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-2347306bdf11e71d3b103785c1f8574bc23f977a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-2347306bdf11e71d3b103785c1f8574bc23f977a-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 164b402b223943ffd1a6b02eafb2aefbf76f0711:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-420e65ba0e93be9df3861bd6916e197bf2a28232-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-420e65ba0e93be9df3861bd6916e197bf2a28232-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-420e65ba0e93be9df3861bd6916e197bf2a28232-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-420e65ba0e93be9df3861bd6916e197bf2a28232-x86.apk)

**Review by m-kuhn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 54359dc70bef244ad84eb900a72ef41c3fd3b3c2:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-4cd318dd138a5b8fdfe49a3bc7d08b51272371c7-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-4cd318dd138a5b8fdfe49a3bc7d08b51272371c7-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-4cd318dd138a5b8fdfe49a3bc7d08b51272371c7-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-4cd318dd138a5b8fdfe49a3bc7d08b51272371c7-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f34cbe11fbf0d5b1230933be0f646f5ead817abf:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-3ac770ea9d0c50f056e3dea31a8cd3956aa4221b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-3ac770ea9d0c50f056e3dea31a8cd3956aa4221b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-3ac770ea9d0c50f056e3dea31a8cd3956aa4221b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-3ac770ea9d0c50f056e3dea31a8cd3956aa4221b-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e16686a133ae8ebf011bc706ab84ded68f92a7d4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-67524ffc60cbdf4809bcc464a6b3e52d3b0a67ab-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-67524ffc60cbdf4809bcc464a6b3e52d3b0a67ab-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-67524ffc60cbdf4809bcc464a6b3e52d3b0a67ab-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-67524ffc60cbdf4809bcc464a6b3e52d3b0a67ab-x86.apk)

File: vcpkg/overlay/sqlite3/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.10)
```
Comment (m-kuhn) [inline vcpkg/overlay/sqlite3/CMakeLists.txt:1]: Patch for sqlite3 merged upstream, can be removed here as soon as we bump the submodule version

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 005ab991ef58c9d374cc754483bf1c456559207f:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-ca167698dcb1cb7a1c34e118362f511d12a6a43b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-ca167698dcb1cb7a1c34e118362f511d12a6a43b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-ca167698dcb1cb7a1c34e118362f511d12a6a43b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-ca167698dcb1cb7a1c34e118362f511d12a6a43b-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for cb0ff9fbf121c0c77789c647a780115a374f5f57:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-871e4d47951e7bf6c88b85faa2acfc3aeface3e8-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-871e4d47951e7bf6c88b85faa2acfc3aeface3e8-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-871e4d47951e7bf6c88b85faa2acfc3aeface3e8-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-871e4d47951e7bf6c88b85faa2acfc3aeface3e8-x86.apk)

**Comment by nirvn** [comment]
@m-kuhn , what should we do with this?

**Comment by m-kuhn** [comment]
a) wait a little longer
b) rebase the branch on our repo and move ahead
Preference?

**Comment by nirvn** [comment]
Maybe we go for (b) and drop a comment in the vcpkg PR :)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9679dafa9fe4e27df7a07de11cd6c76fd220da64:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-09c835b981fe10a2b7d952001eaebdc5fa2fac50-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-09c835b981fe10a2b7d952001eaebdc5fa2fac50-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-09c835b981fe10a2b7d952001eaebdc5fa2fac50-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2363-09c835b981fe10a2b7d952001eaebdc5fa2fac50-x86.apk)

**Pull request merged**